app/assets/javascripts/cookiesalert/cookies_alert.js

```diff
@@ -9,30 +9,33 @@ var cookies = {
       cookies.btn = obj.btn || cookies.btn
       cookies.expires = obj.expires || cookies.expires
     }
- 
-    if (Cookies.get('cookies_alert') === undefined) {
+
+    if (Cookies.get('cookies_consent') === undefined) {
       $(cookies.div).css({ visibility : 'visible' })
       $(cookies.btn).click(function () {
         cookies.accept();
+        Cookies.set('cookies_consent', 'true');
         setNonTechnicalCookies();
       })
       $(cookies.denyBtn).click(function () {
         cookies.remove();
-        $(cookies.div).css({ visibility : 'visible' });
+        Cookies.set('cookies_consent', 'false');
+        $(cookies.div).css({ visibility : 'hidden' });
       })
     } else {
       $(cookies.denyBtn).click(function () {
         cookies.remove();
+        Cookies.set('cookies_consent', 'false');
```

Comment:
this is already being done in `remove` which is called in L27. No need to do it again

Reply:
Removed